### PR TITLE
Fixes in `find_super_nodes` and `find_big_family` command of keeper-client

### PR DESCRIPTION
**Pull request by davenger**
### Changelog category (leave one):
- Bug Fix (user-visible misbehavior in an official stable release)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fixes in `find_super_nodes` and `find_big_family` command of keeper-client:
- do not fail on ZNONODE errors
- find super nodes inside super nodes
- properly calculate subtree node count 

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

<details>
    <summary>Modify your CI run</summary>

**NOTE:** If your merge the PR with modified CI you **MUST KNOW** what you are doing
**NOTE:** Checked options will be applied if set before CI RunConfig/PrepareRunConfig step

#### Include tests (required builds will be added automatically):
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Unit tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with Analyzer
- [ ]  All with Azure
- [ ]  Add your option here

#### Exclude tests:
- [ ]  Fast test
- [ ]  Integration Tests
- [ ]  Stateless tests
- [ ]  Stateful tests
- [ ]  Performance tests
- [ ]  All with ASAN
- [ ]  All with TSAN
- [ ]  All with MSAN
- [ ]  All with UBSAN
- [ ]  All with Coverage
- [ ]  All with Aarch64
- [ ]  Add your option here

#### Extra options:
- [ ]  do not test (only style check)
- [ ]  disable merge-commit (no merge from master before tests)
- [ ]  disable CI cache (job reuse)

#### Only specified batches in multi-batch jobs:
- [ ]  1
- [ ]  2
- [ ]  3
- [ ]  4

</details>

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit 4de7773783d8d31b36f40e2951358b67f30c946b with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/63862/4de7773783d8d31b36f40e2951358b67f30c946b/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: programs/keeper-client/Commands.cpp [hunk 1]
```diff
@@ -310,31 +321,34 @@ bool FindBigFamily::parse(IParser::Pos & pos, std::shared_ptr<ASTKeeperQuery> &
...
+
+    Strings children;
+    auto status = client->zookeeper->tryGetChildren(path, children);
+    if (status == Coordination::Error::ZNONODE)
+        return 0;
+    else if (status != Coordination::Error::ZOK)
+        throw DB::Exception(DB::ErrorCodes::KEEPER_EXCEPTION, "Error {} while getting children of {}", status, path.string());
+
+    for (auto & child : children)
+        nodes_in_subtree += traverse(path / child, client, result);
```
Comment (nikitamikhaylov) [inline programs/keeper-client/Commands.cpp:337]: For future: we can traverse the tree in parallel using multiple threads.

**Comment by nikitamikhaylov** [comment]
I forgot what do these commands do, please correct me if I'm wrong, but `find_super_nodes` finds nodes with the biggest number of children and `find_big_family` finds the nodes with the biggest number of nodes in the subtree (but looks like the root will be always on the top).

**Review by nikitamikhaylov — APPROVED** [review]

**Comment by davenger** [comment]
> I forgot what do these commands do, please correct me if I'm wrong, but `find_super_nodes` finds nodes with the biggest number of children and `find_big_family` finds the nodes with the biggest number of nodes in the subtree (but looks like the root will be always on the top).

Exactly!
And for `find_big_family` you are right that the path that you specify (or root path by default) will always be the top one.

**Pull request merged**